### DISCUSSION TITLE
Disabled options when product out of stock

### DISCUSSION THREAD

**Discussion by pk-pankaj** [breezefront/community#270] [3. Integrations]
Hi @vovayatsyuk 

Product options are not being disabled when a product is out of stock with Breeze, it's working fine with Luma.

So have you not integrated this in Breeze?

![MicrosoftTeams-image](https://github.com/breezefront/community/assets/19511407/f5e1f5d2-69e0-4e20-9844-df2190f22188)

**Comment by vovayatsyuk** [comment]
Looks like a bug. Will see what it wrong here.

Update: Fixed in https://github.com/breezefront/module-breeze/commit/eb325dfd1fe9629a11350cada6f53bc1629a6c1f
